assets/i18n/ast.json

```diff
@@ -15,8 +15,8 @@
 	"I18N_CREATE_EXPLORATION_UPLOAD": "Xubir",
 	"I18N_CREATE_NO_THANKS": "Non, gracies",
 	"I18N_CREATE_YES_PLEASE": "¡Sí, por favor!",
-	"I18N_DASHBOARD_MY_COLLECTIONS": "Les mios colecciones",
-	"I18N_DASHBOARD_MY_EXPLORATIONS": "Les mios esploraciones",
+	"I18N_DASHBOARD_COLLECTIONS": "Les mios colecciones",
```

Comment:
I agree, but the difficult part is finding translators for lots of languages at short notice (if we don't want the released code to have unexpected gaps). I don't think there's a straightforward way around that no matter how much we try to automate things, so the best thing I can think of is to do a best-effort update in the interim. Fortunately, we don't often change existing strings, so this doesn't happen often, but this is part of what designing for an international audience involves.